src/utils/hooks/useRouter.tsx

```diff
@@ -34,6 +35,17 @@ interface RouterResult {
   location: any
 }
 
+const replaceHifenKeys = (parsedQuery: { [key: string]: any }) => {
+  if (Object.keys(parsedQuery).length === 0) return parsedQuery
+  const replacedHifenKeys = Object.keys(parsedQuery).map((key) => {
+    const convertedKey = key.replace(/-([a-z])/g, function (m, w) {
```

Comment:
Could you give `m` and `w` more descriptive names please?

Reply:
Carl, I don't think this needs to be specified since it's just a utility minor function. I'm not entirely sure on what are the 2 properties there, I think it's ok to leave it like that.